### DISCUSSION TITLE
Support For Safari 15

### DISCUSSION THREAD

**Discussion by tajhans** [monkeytypegame/monkeytype#1666] [Ideas] [answered]
I was on Monkey type using the Safari 15 beta and I noticed that the tab bar was the wrong color. I don't know if support was added for all the themes but I think there should be more support for the latest features. I could work on that or someone else. I doesn't require that much work so whoever works on it could finish pretty quick. Safari already defaults the colors so we don't have to do it but sometimes the color doesn't match all that well.

**Comment by tajhans** [comment, marked as answer]
Okay someone already did it.

**Reply by Miodec** [reply]
That couldve been me

**Reply by tajhans** [reply]
Oh okay. Thanks.
